### ISSUE TITLE
Combined tabs_on_bottom.css and tabs_on_bottom_menubar_on_top_patch.css

### ISSUE THREAD

**Issue by psxlover** [MrOtherGuy/firefox-csshacks#143]
Hallo,

after quite some time having a broken userChrome, I spend my evening yesterday trying to update my css for tabs on bottom to work with the latest firefox version. I didn't find your tabs_on_bottom yesterday, so I started building mine based on another one for firefox 89+ I found online. I think I may have stumbled on your sometime in the past, but I guess I only tried `tabs_on_bottom.css` without `window_control_placeholder_support.css` at that time so I never really used it.

Anyway, in your implementation, you have two different files for when menubar is enabled vs when it isn't. When using `tabs_on_bottom.css` if you press `alt` button to bring the menubar it appears on a completely wrong place. Perhaps you could combine the two to create a better one. You can find my attempt in https://gist.github.com/psxlover/938dc43acf5130eba47a68da5596f517

**Comment by MrOtherGuy** [comment]
Yeah, Ideally there wouldn't be need for multiple files. But I've chosen the way things are because I figured style compatibility is more important than correct position of menubar in all scenarios. For example, there's no way to make your style work with multi-row tabs, or tabs being hidden when only one tab is open. Besides, there's other ways to show menubar when triggered with Alt like overlay_menubar.css so incorrect location isn't such a big deal IMO.

I mean, your style works very well for what it does. And it's one of cleaner ones I have seen that use that kind of method to move tabs below nav-bar so definitely good job. But there's pros and cons for each.
